src/frameworks/ccpa_from_us_privacy_string.js

```diff
@@ -0,0 +1,47 @@
+const FrameworkBase = require('./base');
+
+/**
+ * Implements usprivacy string framework
+ * for more information on the US Privacy string see:
+ * https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/master/CCPA/US%20Privacy%20String.md#us-privacy-string-format
+ */
+class CCPAFromUSPrivacyString extends FrameworkBase {
+  constructor() {
+    super();
+    this.usPrivacyString = '';
+  }
```

Comment:
Is this also missing a `name`? (Or maybe I'm misunderstanding the ctor setups...)

Reply:
It will inherit this from the constructor, so unless we need to override it, all the subclasses get the `name` `getter`.